index.html

```diff
@@ -179,6 +324,145 @@ <h2 class="about-me-header">Software Developer</h2>
           <p class="work-paragraph">Work</p>
         </div>
       </div>
+      <section class="my-recent-work">
+        <h2 class="recent-work-header">MY RECENT WORK</h2>
+
+        <div class="recent-work-div">
+          <div class="recent-work-demo-wrapper">
+            <img
+              src="./images/.png"
+              class="recent-work-demo"
+              alt="my recent work"
+            />
+          </div>
+          <article class="recent-work-article">
+            <h3 class="recent-work-article-header">Multi-post-story</h3>
+            <p class="recent-work-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul
+              class="recent-work-ul display-flex justify-content-space-between"
+            >
+              <li class="recent-work-ul-li">css</li>
+              <li class="recent-work-ul-li">html</li>
+              <li class="recent-work-ul-li">boostrap</li>
+              <li class="recent-work-ul-li border-none">Ruby</li>
+            </ul>
+            <button
```

Comment:
<img width="388" alt="Screenshot 2021-10-07 at 06 35 32" src="https://user-images.githubusercontent.com/53398818/136325922-82432a00-0b77-407d-93bf-8001549b03e4.png">
The buttons should have the active, hover and pressed states